core/ui/prime/components/src/lib/autocomplete/autocomplete.component.html

```diff
@@ -2,14 +2,12 @@
   [suggestions]="filteredOptions"
   field="value"
   [dropdown]="dropdown"
-  (onSelect)="onItemSelected(value)"
-  (onUnselect)="onItemUnselected(value)"
-  (onDropdownClick)="onSearched($event)"
-  (onBlur)="onBlur($event)"
+  (onSelect)="onItemSelected($event)"
+  (onUnselect)="onItemUnselected($event)"
   (completeMethod)="onSearched($event)"
   (onClear)="onCleared()"
-  dropdownMode="current"
-  [(ngModel)]="value"
+  dropdownMode="blank"
```

Comment:
note to self: doublecheck change

Reply:
Value needs to be blank to prevent the list to be filtered on opening if a value is selected. This behavior isn't user friendly since you first have to remove the text from the dropdown before the other values are shown